seqdb/db.js

```diff
@@ -0,0 +1,17 @@
+var Sequelize = require('sequelize');
+
+var sequelize = new Sequelize('excellen_hr_demo', 'sumit', '123', { dialect: 'mysql' });
```

Comment:
db details should not be hard coded. it should either come from config file or .env file. discuss with saurabh how to do this or google about it